clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/ArraysExtensions.scala

```diff
@@ -1,41 +1,40 @@
 package org.broadinstitute.clio.transfer.model.arrays
 
+import org.broadinstitute.clio.transfer.model.FileExtensions
+
 /**
   *  Extensions for files in the arrays index.
   */
-object ArraysExtensions {
+object ArraysExtensions extends FileExtensions {
 
   val BpmExtension = ".bpm"
 
-  val CsvExtension = ".csv"
-
-  val DictExtension = ".dict"
-
   val EgtExtension = ".egt"
 
-  val FastaExtension = ".fasta"
+  val IdatExtension = ".idat"
 
-  val FaiExtension = ".fai"
+  val GrnIdatExtension = s"_Grn$IdatExtension"
```

Comment:
It looks like this is only used in tests. If so, it should be defined in a test class.

Reply:
It's used in DeliverArraysExecutor now.